docs/send-data/kubernetes/logs.md

```diff
@@ -0,0 +1,553 @@
+- Use `^Error` expression for all remaining `stderr` logs which are not in `lorem` namespace
+- Use `^\\[?\\d{4}-\\d{1,2}-\\d{1,2}.\\d{2}:\\d{2}:\\d{2}` expression for all remaining logs
+
+:::note
+Logs which match multiple conditions are processed only by the first match.
+:::
+
+[expr]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.90.1/pkg/stanza/docs/types/expression.md
+
+### Container Log format
+
+There are three log formats available: `fields`, `json_merge` and `text`. `fields` is the default.
+
+You can change it by setting:
+
+```yaml
+sumologic:
+  logs:
+    container:
+      format: fields
+```
+
+We're going to demonstrate the differences between them on two example log lines:
+
+1. A plain text log
+
+   ```text
+   2007-03-01T13:00:00Z I am a log line
+   ```
+
+1. A JSON log
+
+   ```json
+   { "log_property": "value", "text": "I am a json log" }
+   ```
+
+#### `json` log format
+
+`json` log format is an alias for `fields` log format.
+
+See [`fields` log format](#fields-log-format)
+
+#### `fields` log format
+
+Logs formatted as `fields` are wrapped in a JSON object with additional properties, with the log body residing under the `log` key.
+
+For example, log line 1 will show up in Sumo Logic as:
+
+```javascript
+{
+  log: "2007-03-01T13:00:00Z I am a log line",
+  stream: "stdout",
+  timestamp: 1673627100045
+}
+```
+
+If the log line contains json, as log line 2 does, it will be displayed as a nested object inside the `log` key:
+
+```javascript
+{
+  log: {
+    log_property: "value",
+    text: "I am a json log"
+  },
+  stream: "stdout",
+  timestamp: 1673627100045
+}
+```
+
+#### `json_merge` log format
+
+`json_merge` is identical to `fields` for non-JSON logs, but behaves differently for JSON logs. If the log is JSON, it gets merged into the
+top-level object.
+
+Log line 1 will show up the same way as it did for `fields`:
+
+```javascript
+{
+  log: "2007-03-01T13:00:00Z I am a log line",
+  stream: "stdout",
+  timestamp: 1673627100045
+}
+```
+
+However, the attributes from log line 2 will show up at the top level:
+
+```javascript
+{
+  log: {
+    log_property: "value",
+    text: "I am a json log"
+  },
+  stream: "stdout",
+  timestamp: 1673627100045
+  log_property: "value",
+  text: "I am a json log"
+}
+```
+
+#### `text` log format
+
+The `text` log format sends the log line as-is without any additional wrappers.
+
+Log line 1 will therefore show up as plain text:
+
+```text
+2007-03-01T13:00:00Z I am a log line
+```
+
+Whereas log line 2 will be displayed as JSON:
+
+```javascript
+{
+  log_property: "value",
+  text: "I am a json log"
+}
+```
+
+If you want to send metadata along with an unstructured log record, you have to use resource level attributes, because record level attributes are going to be
+removed before sending log to Sumo Logic. Please see [Mapping OpenTelemetry concepts to Sumo Logic][mapping] for more details.
```

Comment:
```suggestion
If you want to send metadata along with an unstructured log record, you have to use resource level attributes, because record level attributes are going to be removed before sending log to Sumo Logic. See [Mapping OpenTelemetry concepts to Sumo Logic][mapping] for more details.
```